### ISSUE TITLE
No such module MetalPerformanceShaders 

### ISSUE THREAD

**Issue by lutyyy** [hollance/Forge#13]
I have this error issue. Can you have a look? thanks

**Comment by austingg** [comment]
you should provide more information. 
on iOS10, MPS can only develop on device, not simulator.

**Comment by lutyyy** [comment]
Oh I see. I let me try again, i am update to ios10.3.2
by the way, can i5s run success?

**Comment by hollance** [comment]
MetalPerformanceShaders needs a device with an A8 or higher processor, so a 5s won't work (the minimum device is an iPhone 6). You also cannot run in the simulator. Xcode gives this error if you try to run on the simulator (or I guess on a device that doesn't support MPS).
